lang/en.js

```diff
@@ -0,0 +1,25 @@
+/* eslint quotes: 0 */
+
+export default {
+	"components.alert.alert.close": "Close Alert",
```

Comment:
FYI - Serge has some capability to accept translation hints that will aid the translator.  With the `parse_js` parser, if the key-value pair line ends with a comment, it will be parsed and passed as the hint for the Serge translators.
```
"components.tabs.tabs.scroll.next": "Scroll Forward", // text for button that scrolls next set of tabs into view
```
Most of our English terms in core are hopefully fairly self-explanatory though.

Reply:
Ah cool that's good to know - thanks!